extensions/amp-analytics/amp-analytics.md

```diff
@@ -685,6 +685,49 @@ Referrer policy is only available for `image` transport. If `referrerPolicy: no-
 }
 ```
 
+#### Linkers
+The `linkers` configuration object is used to enable cross domain ID syncing. `amp-analytics` will use this configuration to create an unique query parameter that can be used to join sessions. This parameter will then be appended to the specified outgoing links.
```

Comment:
... `amp-analytics` uses this configuration to create a "linker string"  which gets appended to the specified outgoing links on the page as URL param. When user clicks on one of those links, the destination page then can read the linker string from URL param to perform ID syncing. This is typically used to join user session across AMP proxy domain and publisher domain.